owncloudApp/src/main/java/com/owncloud/android/presentation/ui/settings/fragments/SettingsLogsFragment.kt

```diff
@@ -38,26 +38,26 @@ class SettingsLogsFragment : PreferenceFragmentCompat() {
 
     private var prefEnableLogging: SwitchPreferenceCompat? = null
     private var prefHttpLogs: CheckBoxPreference? = null
-    private var prefLogsView: Preference? = null
+    private var prefBrowse: Preference? = null
```

Comment:
As I suggested in the test, I think that this naming is not as good as it was before. Here our idea is to open the logs view and check the latest logs directly.